service/src/test/java/greencity/mapping/ShortHabitDtoMapperTest.java

```diff
@@ -0,0 +1,35 @@
+package greencity.mapping;
+
+import greencity.dto.habit.ShortHabitDto;
+import greencity.entity.Habit;
+import greencity.entity.HabitTranslation;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.Collections;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ShortHabitDtoMapperTest {
+
+    private ShortHabitDtoMapper mapper;
+    private final Long id = 1L;
+    private final String habitTranslationName = "Test Habit";
+    private final HabitTranslation habitTranslation = HabitTranslation.builder().name(habitTranslationName).build();
+
+    private final Habit habit = Habit.builder()
+        .id(id)
+        .habitTranslations(Collections.singletonList(habitTranslation))
+        .build();
+
+    @BeforeEach
+    void setUp() {
+        mapper = new ShortHabitDtoMapper();
+    }
+
+    @Test
+    void convert_ShouldMapHabitToShortHabitDto() {
```

Comment:
In one of the meetings, we decided to use a naming strategy without underscore.
It'd be better to remove the underscore.

New naming strategy for testing, you can see on the project wiki https://github.com/ita-social-projects/GreenCity/wiki/Naming-Conventions

Reply:
Thanks for saying that. I corrected it.